packages/venia-ui/lib/components/PageLoadingIndicator/indicator.css

```diff
@@ -0,0 +1,17 @@
+.root {
+    justify-self: start;
+}
+
+.indicator {
+    animation: spin 0.75s linear infinite;
```

Comment:
Out of interest why do you use base 2?

Reply:
It's more evenly divisible, which helps when figuring out what a given increment should be. It's the same reason most grid systems are 12-column or 16-column instead of 10-column.

```
# easy to keep dividing by 2
16 => 8 => 4 => 2 = > 1

# becomes increasingly harder to do quick math
10 => 5 => 2.5 => 1.25 => 0.625
```

Reply:
Ah yeah, that makes a lot of sense, thanks for the explanation